src/transformers/typescript.ts

```diff
@@ -84,6 +84,24 @@ const TS_TRANSFORMERS = {
   before: [importTransformer],
 };
 
+const TS2552_REGEX = /Cannot find name '\$([a-zA-Z0-9_]+)'. Did you mean '([a-zA-Z0-9_]+)'\?/i;
+function isValidSvelteReactiveValueDiagnostic(
+  filename: string,
+  diagnostic: any,
+): boolean {
+  if (diagnostic.code !== 2552) return true;
+
+  /** if the importee is not a svelte file, do nothing */
+  if (!isSvelteFile(filename)) return true;
+
+  /** if error message doesn't contain a reactive value, do nothing */
+  if (!diagnostic.messageText.includes('$')) return true;
+
+  const matches = diagnostic.messageText.match(TS2552_REGEX);
+
+  return !(matches[1] && matches[2] && matches[1] === matches[2]);
```

Comment:
Can you replace these with a destructured array? It's simpler to grasp what the `matches[1]` and `matches[2]` mean.

Reply:
Got it! Just made those changes in this [commit](https://github.com/kaisermann/svelte-preprocess/pull/126/commits/a9baf041e5ffb3003a6acbffa4279708a7f46f6a)